packages/frontend-api/src/Model/Mutation/Login/LoginMutation.php

```diff
@@ -63,9 +64,11 @@ public function login(Argument $argument): array
             throw new UserError('Log in failed.');
         }
 
+        $deviceId = Uuid::uuid4()->toString();
+
         return [
-            'accessToken' => $this->tokenFacade->generateAccessTokenByCustomerUser($user),
-            'refreshToken' => $this->tokenFacade->createRefreshTokenAsString($user),
+            'accessToken' => $this->tokenFacade->generateAccessTokenByCustomerUserAndDeviceId($user, $deviceId),
```

Comment:
why method for generating of access token is named `generateAccessTokenByCustomerUserAndDeviceId` and for refresh token it is `createRefreshTokenAsString` this should be unified - what do you mean?

Reply:
9f1ba3e57fe0dfb092488c9034383132c8f8d419